src/main/java/org/monarchinitiative/phenopacket2prompt/output/PromptGenerator.java

```diff
@@ -104,9 +105,10 @@ default String createPrompt(PpktIndividual individual) {
                     createPromptWithoutHeader(individual));
         }
         else {
-            return String.format("%s%s",
+            return String.format("%s%s%s",
                     getHeader(),
-                    createPromptWithoutHeader(individual));
+                    createPromptWithoutHeader(individual),
```

Comment:
Future implementation would like this to pull from a file as well that has place holders for the different types of information so we can create the prompt manually too.